zombienet_tests/async_backing/async-backing.feature

```diff
@@ -0,0 +1,31 @@
+Description: Async Backing Version Test
+Network: ./async-backing-test.toml
+Creds: config
+
+# General
+alice: is up
+bob: is up
+
+# Check authority status 
+alice: reports node_roles is 4
+bob: reports node_roles is 4
+
+# Check peers
+alice: reports peers count is at least 2 within 20 seconds
```

Comment:
peer count should be 3, since we also have a collator.

Reply:
There are 3 nodes in total, so every node can have at most 2 peers.

Reply:
🤦‍♂️ yeah, you are right